### ISSUE TITLE
test fused batch norm

### ISSUE THREAD

**Issue by Highlife1911** [HPI-DeepLearning/SegMed#31]
see https://www.tensorflow.org/api_docs/python/tf/contrib/layers/batch_norm

**Comment by Highlife1911** [comment]
it is like 10% faster so i use it now.
